src/libraries/sendtohelixhelp-wasm.proj

```diff
@@ -0,0 +1,221 @@
+<Project>
```

Comment:
Please rename these files to `.targets` file since it is no longer an `invokable` project, it is more a file with targets that gets imported by the invokable `sendtohelixhelp.proj`

Also, rather than including `help` on these projects can we name them something like `sendtohelix-wasm.targets`?

Reply:
re:targets, 👍 

re:-help, I was trying to be clear that this is used by `sendtohelixhelp.proj`, but renaming is fine too.